### PR TITLE
Autoload

### PR DESCRIPTION
**Pull request by philipjackson**
# Description

Added a toy dataset of 20 RGB png images, and some code in `examples/src/index.tsx` to autoload them and pass them into `UserInterface` as props. Right now there is a simple interface for passing in tiles as an array of `{name, label, thumbnail}` objects. For now I've gone with base64 for the images, since `ImageBitmap` would require us to spawn a new canvas for every thumbnail tile which sounded a bit mad, and I know etebase likes strings. 

There's currently no distinction between thumbnails and actual images, but I'm guessing CURATE shouldn't need the all the full images upfront anyway.

relates #1 #4

# Dependency changes

material-ui

# Testing

N/A

# Documentation

N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+
+interface Props {
+  tiles: Array<Tile>;
+}
+
+export const UserInterface = (props: Props): React.ReactElement => (
+  <div>
+    <h1>CURATE</h1>
+    <Grid container spacing={3}>
+      {props.tiles.map((tile, index) => (
```
Comment (ChasNelson1990) [inline src/index.tsx:18]: Replace with (I think):
```typescript
      {props.tiles.map((tile) => (
        <Grid item xs={1} key={tile.id}>
          <img
            height={128}
            src={`data:image/png;base64,${tile.thumbnail}`}
            alt={tile.name}
          />
        </Grid>
      ))}
```

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
instead of adding these images, which aren't our images, does it make more sense to have a small script that downloads them directly from their source?

File: src/index.tsx (hunk 1 above)
Comment (philipjackson) [reply src/index.tsx]: Done

**Comment by ChasNelson1990** [comment]
> instead of adding these images, which aren't our images, does it make more sense to have a small script that downloads them directly from their source?

Did you think any further about this one @philipjackson ?

**Comment by philipjackson** [comment]
> > instead of adding these images, which aren't our images, does it make more sense to have a small script that downloads them directly from their source?
> 
> Did you think any further about this one @philipjackson ?

I probably _could_, but the source is an 840MB zip file from BBBC. The images I've saved to the repo were created by stacking three of the images in that zip and then slicing into 20 equally sized tiles in numpy.

**Review by ChasNelson1990 — DISMISSED** [review]

**Review by ChasNelson1990 — DISMISSED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Review by cooper667 — APPROVED** [review]

**Pull request merged**